packages/shared/src/utils/cfd/cfd.ts

```diff
@@ -40,7 +41,7 @@ type TGetCFDAccountKey = TGetAccount & {
 // sub_account_type financial_stp only happens in "financial" market_type
 export const getCFDAccountKey = ({ market_type, sub_account_type, platform, shortcode }: TGetCFDAccountKey) => {
     if (market_type === 'all') {
-        return platform === CFD_PLATFORMS.DERIVEZ ? 'derivez' : 'dxtrade';
+        return platform === CFD_PLATFORMS.DERIVEZ ? 'derivez' : platform === 'ctrader' ? 'ctrader' : 'dxtrade';
```

Comment:
Can use the Switch case or if blocks
```suggestion
        switch(platform){
        case `${CFD_PLATFORMS.DERIVEZ}`:
          return 'derivez'
         case `${CFD_PLATFORMS.CTRADER}`:
          return 'ctrader'
        default:
         return 'dxtrade'
        
```